components/file_upload/file_upload.jsx

```diff
@@ -457,7 +457,9 @@ export default class FileUpload extends PureComponent {
     }
 
     handleMaxUploadReached = (e) => {
-        e.preventDefault();
+        if (e) {
```

Comment:
This extra condition check is not necessary but added anyway for protection in the future.